### DISCUSSION TITLE
Sync?

### DISCUSSION THREAD

**Discussion by MitchWagnerSD** [chhoumann/quickadd#590] [Q&A] [answered]
Do QuickAdd settings sync between devices on which Obsidian is installed? 

I have QuickAdd running on an iPad Mini, Mac, iPhone and iPad Air. I changed a Capture on my iPad mini, but the Capture item hasn't changed on my iPhone or iPad Air (I haven't checked the Mac yet). 

Are changes to QuickAdd Captures supposed to sync between devices?

I'm using the Obsidian Sync service and have it set to sync everything between all my devices, including installed community plugins, which is also supposed to sync settings for community plugins. 

How can I solve this problem?

Thanks!

**Comment by MitchWagnerSD** [comment, marked as answer]
I think I've figured out the problem. If you change a setting on one device, you need to restart Obsidian on the other devices for the setting to sync. 

I assume on iPhones and iPads, that means force-quitting Obsidian and restarting it, or restarting the device.

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that. Your diagnosis is exactly right, and it still holds on current QuickAdd: Obsidian Sync does sync the settings file, but plugins only read their settings when they load. So after a change syncs over, the other device needs an Obsidian restart (or toggling QuickAdd off and on under Community plugins) to pick it up. Marking your comment as the answer.
